ts/Grid/Core/Querying/QueryingController.ts

```diff
@@ -91,7 +98,7 @@ class QueryingController {
     public async proceed(force: boolean = false): Promise<void> {
         if (
             force ||
-            this.sorting.shouldBeUpdated ||
+            this.shouldBeUpdated ||
             this.filtering.shouldBeUpdated
```

Comment:
Note to remember this later - `shouldBeUpdated` should be removed from filtering, a global flag for querying is enough.